app/assets/stylesheets/pageflow/editor/files_explorer.css.scss

```diff
@@ -33,6 +33,12 @@
 
   .entries_panel {
     width: 20%;
+
+    li.blank_slate {
+      text-align: center;
+      padding: 10px 5px 20px 5px;
+      color: #888;
```

Comment:
Color literals like `#888` should only be used in variable declarations; they should be referred to via variable everywhere else.